djangocms_blog/views.py

```diff
@@ -22,6 +22,16 @@
 class BaseBlogView(AppConfigMixin, ViewUrlMixin):
     model = Post
 
+    def optimize(self, qs):
+        """
+        Apply select_related / prefetch_related to optimize the view queries
+        :param qs: queryset to optimize
+        :return: optimized queryset
+        """
+        return qs.select_related('app_config').prefetch_related(
+            'translations', 'categories', 'categories__translations', 'categories__app_config'
+        )
+
```

Comment:
I chose to keep two copies because I want to be able to evolve those indipendently as plugins and views may have different needs. This is just a first stab at the optimization